diff -urN libgsf-1.14.5/configure.in libgsf-1.14.5.new/configure.in
--- libgsf-1.14.5/configure.in	2007-07-10 12:33:55.000000000 +0200
+++ libgsf-1.14.5.new/configure.in	2007-07-26 01:04:32.000000000 +0200
@@ -401,17 +401,7 @@
 dnl The following conditional is set in AM_GCONF_SOURCE_2.
 dnl Because we may skip its execution, we have to set a default here.
 AM_CONDITIONAL([GCONF_SCHEMAS_INSTALL], [false])
-
-dnl Don't publish the GCONFTOOL variable, AM_GCONF_SOURCE_2 has ``gconftool-2'' hardwired.
-m4_pattern_allow([^AM_GCONF_SOURCE_2$])
-GCONFTOOL=
-AC_CHECK_PROG([GCONFTOOL], [gconftool-2], [gconftool-2], [no])
-if test "x$GCONFTOOL" = "xno"; then
-    AC_MSG_WARN([thumbnailer will not be built, unable to find gconftool-2])
-else
-    AM_GCONF_SOURCE_2
-fi
-AM_CONDITIONAL(WITH_GCONF, test "x$GCONFTOOL" != "xno")
+AM_CONDITIONAL(WITH_GCONF, false)
 
 LDFLAGS="-no-undefined $LDFLAGS"
 
