--- libgsf-1.14.9/configure.in.orig	2008-08-31 15:31:53.490120063 +0200
+++ libgsf-1.14.9/configure.in	2008-08-31 16:07:35.406119225 +0200
@@ -85,8 +85,6 @@
 dnl Modules required for libgsf-gnome
 libgsf_gnome_reqs="
     $common_reqs
-    gnome-vfs-2.0 >= 2.2.0
-    gnome-vfs-module-2.0 >= 2.2.0
 "
 PKG_CHECK_MODULES(LIBGSF, $libgsf_reqs)
 
--- libgsf-1.14.9/gsf-gnome/gsf-input-gnomevfs.h.orig	2008-04-22 13:19:46.000000000 +0200
+++ libgsf-1.14.9/gsf-gnome/gsf-input-gnomevfs.h	2008-09-15 01:17:01.000000000 +0200
@@ -22,6 +22,7 @@
 #ifndef GSF_INPUT_GNOMEVFS_H
 #define GSF_INPUT_GNOMEVFS_H
 
+#include <gsf-config.h>
 #include <gsf/gsf-input.h>
 #ifndef LIBGSF_GNOMEVFS_VIA_GIO
 /*
@@ -29,6 +30,8 @@
  * want to avoid it.
  */
 #include <libgnomevfs/gnome-vfs.h>
+#else
+#define GnomeVFSURI void
 #endif
 
 G_BEGIN_DECLS
--- libgsf-1.14.9/gsf-gnome/gsf-output-gnomevfs.h.orig	2008-07-03 16:44:51.000000000 +0200
+++ libgsf-1.14.9/gsf-gnome/gsf-output-gnomevfs.h	2008-09-15 01:18:03.000000000 +0200
@@ -22,6 +22,7 @@
 #ifndef GSF_OUTPUT_GNOMEVFS_H
 #define GSF_OUTPUT_GNOMEVFS_H
 
+#include <gsf-config.h>
 #include <gsf/gsf-output.h>
 #ifndef LIBGSF_GNOMEVFS_VIA_GIO
 /*
@@ -29,6 +30,8 @@
  * want to avoid it.
  */
 #include <libgnomevfs/gnome-vfs.h>
+#else
+#define GnomeVFSURI void
 #endif
 
 G_BEGIN_DECLS
