--- trunk/gsf/gsf-output-csv.c	2009/01/26 18:47:59	1038
+++ trunk/gsf/gsf-output-csv.c	2009/01/30 16:21:08	1039
@@ -182,7 +182,7 @@
 	static GType gsf_output_csv_quoting_mode_type = 0;
 
 	if (gsf_output_csv_quoting_mode_type == 0) {
-		GEnumValue const values[] = {
+		static GEnumValue const values[] = {
 		{ GSF_OUTPUT_CSV_QUOTING_MODE_NEVER,  (char *)"GSF_OUTPUT_CSV_QUOTING_MODE_NEVER",  (char *)"never" },
 		{ GSF_OUTPUT_CSV_QUOTING_MODE_AUTO,   (char *)"GSF_OUTPUT_CSV_QUOTING_MODE_AUTO",   (char *)"auto" },
 		{ GSF_OUTPUT_CSV_QUOTING_MODE_ALWAYS, (char *)"GSF_OUTPUT_CSV_QUOTING_MODE_ALWAYS", (char *)"always" },
